src/robot_folders/helpers/option_helpers.py

```diff
@@ -0,0 +1,56 @@
+import click
+
+
+# stolen from https://stackoverflow.com/questions/48391777/nargs-equivalent-for-options-in-click/48394004#48394004
```

Comment:
This is rather problematic from a licensing point of view. Since stackoverflow contents are licensed under CC-BY-SA licenses that would inject a copyleft license in our code. 

I would have to ask you to rewrite that functionality with your own code in order to add that to this package.